content/docs/09.administrator-guide/03.monitoring.md

```diff
@@ -94,19 +94,19 @@ triggers:
       - type: io.kestra.plugin.core.condition.OrCondition
         conditions:
           - type: io.kestra.plugin.core.condition.ExecutionNamespaceCondition
-            namespace: product
+            namespace: company.analytics
             prefix: true
           - type: io.kestra.plugin.core.condition.ExecutionFlowCondition
             flowId: cleanup
-            namespace: system
+            namespace: company.monitoring
```

Comment:
I'd keep this as a system namespace. We want to encourage people to have a system namespace for these type of flows, but it should fall under the company namespace.

```suggestion
            namespace: company.system
```